README.md

```diff
@@ -14,15 +15,26 @@ $ dmenu_tgit
 ```
 
 ## dependencies:
+
 - git
 - find
 - awk
 
+## Instalation
+
+To install `tmenu` you can edit the `Makefile` to match your local setup (`tmenu` is installed into the `/usr/local/bin` by default).
```

Comment:
The name of the package is wrong.
It should be `tgit` instead.

Reply:
silly mistake of mine 😬
Just fixed it!